Formula/nativefier.rb

```diff
@@ -0,0 +1,19 @@
+require "language/node"
+
+class Nativefier < Formula
+  desc "Wrap web apps natively"
+  homepage "https://github.com/jiahaog/nativefier"
+  url "https://registry.npmjs.org/nativefier/-/nativefier-7.0.1.tgz"
+  sha256 "09a22a0d4b5fe85b5075d2a5896bb3648928d0da73c44c61fdbded2bda608d5e"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system bin/"nativefier", "--version"
```

Comment:
Unfortunately, due to its dependency  [node-progress](https://github.com/visionmedia/node-progress), **nativefier** cannot run in non-tty mode such as `brew test`. I cannot add any meaningful test out of tty mode. There are some [pull requests](https://github.com/visionmedia/node-progress/pull/139) waiting for reply to fix this issue. Still, **nativefier** works well in tty mode.